dandi/metadata.py

```diff
@@ -83,3 +88,62 @@ def get_metadata(path):
     meta["nd_types"] = get_neurodata_types(path)
 
     return meta
+
+
+def nwb_to_json(fpath, out_fpath):
+    io = NWBHDF5IO(fpath, 'r')
+    file = File(fpath)
+    io.read()
+    root_builder = io.get_builder(file['general']).parent
+    out_dict = group_builder_to_json(root_builder)
+    return json.dump(out_dict, out_fpath)
+
+
+def print_dataset(dataset):
+    return str(dataset)  # may want to change
```

Comment:
I don't see a point of having this function.  not to say it doesn't print anythning but rather `== str`

Reply:
This is the simplest way I could think of to handle h5py Dataset objects, but I'm thinking we probably want to change this to expose more metadata about the dataset. Something like:
```
{'size': (100,5),
 'dtype': 'int',
 'path': '/acquisiton/ElectricalSeries/data',
 'compression':('gzip', 5)} 
```
so this function is more of a place-holder until we decide on how to do this